components/gitpod-db/src/typeorm/project-db-impl.ts

```diff
@@ -43,4 +43,10 @@ export class ProjectDBImpl implements ProjectDB {
         await repo.save(project);
         return project;
     }
+
+    async findProjectByInstallationId(appInstallationId: string): Promise<Project | undefined> {
```

Comment:
Maybe make this `public`?

```suggestion
    public async findProjectByInstallationId(appInstallationId: string): Promise<Project | undefined> {
```

Reply:
it's public by [default](https://www.typescriptlang.org/docs/handbook/classes.html#public-by-default).

Reply:
Most other code locations have `public` set explicitly, so this seems inconsistent. Not very important though, so resolved for now.